### DISCUSSION TITLE
How to let Number input (q-input) arrow increase/decrease by 0.01

### DISCUSSION THREAD

**Discussion by jianyunli** [quasarframework/quasar#12064] [General - Components / Directives / etc]
I couldn't find documentation on how to change that, the default is 1. I have a floating value, I want to change the delta to 0.01, thanks

**Comment by metalsadman** [comment]
that's native, you will have to implement your own.

**Reply by jianyunli** [reply]
ah, it turns out HTML native steps works
` <q-input  type="number" step="0.01" />`
